gpt4all-chat/embllm.cpp

```diff
@@ -303,6 +303,20 @@ EmbeddingLLM::~EmbeddingLLM()
     m_embeddingWorker = nullptr;
 }
 
+QString EmbeddingLLM::model() const
+{
+    const EmbeddingModels *embeddingModels = ModelList::globalInstance()->installedEmbeddingModels();
+    if (!embeddingModels->count())
+        return QString("Unknown");
```

Comment:
Since this value is directly added to the database, I would prefer to throw an exception here. From a UI standpoint, the user should never be able to add collections to the database with no embedding model selected.

Also, I would prefer if this function were called by LocalDocs::addFolder, and the embedding model were passed as a parameter. Queueing an operation that later checks the state of the UI seems like bad practice.

Reply:
Added code to check.